s/elements/lookpad/element.ts

```diff
@@ -0,0 +1,25 @@
+
+import {html} from "lit"
+import {MagicElement, mixinCss} from "@chasemoskal/magical"
+
+import {styles} from "./style.css.js"
+
+import {property} from "lit/decorators.js"
+import {setupMouseAndTouchMoveListening} from "./utils/setup-mouse-and-touch-move-listening.js"
+
+@mixinCss(styles)
+export class NubLookpad extends MagicElement {
+
+	@property({type: String, reflect: true})
+	name: string = "p"
```

Comment:
i think we should make the name default to `"1"` like the other elements (see real/mouse.ts)